app/src/components/util.tsx

```diff
@@ -147,4 +147,17 @@ export const DetailsValue = styled.p`
   overflow-wrap: anywhere;
   font-size: 0.9em;
   text-align: right;
-`
\ No newline at end of file
+`
+
+export const ClickableText = styled.text`
+  &:hover {
+    cursor: pointer;
+    color: darkgrey;
+  }
+`
+
+export const NonClickableText = styled.text`
```

Comment:
Don't use `text` outside of SVG.

These aren't focusable either. Set `tabindex="0"` on the elements to fix that or use a button and style it to look like text.